routemaster/services/deliver.rb

```diff
@@ -18,6 +18,8 @@ class Deliver
       include Mixins::LogException
       include Mixins::Counters
 
+      FARADAY_USER_AGENT = "routemaster-faraday-#{Faraday::VERSION}".freeze
```

Comment:
Would it make more sense to have a version identifier for routemaster itself?

I don't think we care that much about the version of the library we use to make requests, rather we want to know the origin and behaviour of the requests themselves.

Reply:
While the routemaster-client and routemaster-drain gems have a version, the Routemaster application isn't versioned.